src/components/Graph/Graph.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend, ChartDataLabels);
+
+const Graph = ({ title, xLabels, preData, postData }) => {
+  const yellow = 'rgb(255, 211, 80)';
+  const teal = 'rgb(23, 162, 184)';
+  const textColor = 'rgb(95, 117, 141)';
+  const options = {
+    responsive: true,
+    plugins: {
+      title: {
+        display: true,
+        text: title,
+        font: {
+          size: 18,
+          weight: 'bold',
+        },
+      },
+      legend: {
```

Comment:
I don't think we need a legend. But the bottom bar on the figma shows pre/post.
![image](https://user-images.githubusercontent.com/57849152/162608654-bbb6e84d-b327-49ce-a022-28c5e1af111d.png)